src/test/regress/expected/columnar_chunk_filtering_0.out

```diff
@@ -0,0 +1,421 @@
+--
```

Comment:
why do we add this alternative output file ?

Reply:
There are some minor differences in the way partition plans are explained.